resolwe/flow/management/commands/clean_test_dir.py

```diff
@@ -0,0 +1,53 @@
+""".. Ignore pydocstyle D400.
+
+====================
+Clean test directory
+====================
+
+Command to run on local machine::
+
+    ./manage.py clean_test_dir
+
+"""
+import re
+import os
+import shutil
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from resolwe.flow.managers.protocol import ExecutorFiles
+
+TEST_DIR_REGEX = r'test_\d+'
+
+
+class Command(BaseCommand):
+    """Cleanup files created during testng."""
+
+    help = "Cleanup files created during testng."
+
+    def handle(self, *args, **kwargs):
+        """Cleanup files created during testng."""
+
+        for name in ['DATA_DIR', 'UPLOAD_DIR', 'RUNTIME_DIR']:
+            directory = os.path.abspath(settings.FLOW_EXECUTOR[name])
+
+            for basename in os.listdir(directory):
+
+                fname = os.path.abspath(os.path.join(directory, basename))
+
+                if not os.path.isdir(fname):
+                    continue
+                if not re.match(TEST_DIR_REGEX, basename):
+                    continue
+
+                def handle_error(func, path, exc_info):
+                    """Handle permission errors while removing secrets directory."""
+                    if isinstance(exc_info[1], PermissionError):
+                        # Verify that such woraround can only be applied to secrets dir.
+                        assert os.path.basename(path) == ExecutorFiles.SECRETS_DIR
```

Comment:
Is there any particular reason for checking this?

Reply:
This is to prevent deleting something else than `secrets` dir. There might be a good reason to keep that data. Like some sort of defensive check.